CMakeLists.txt

```diff
@@ -26,7 +27,8 @@ include(CheckSymbolExists)
 include(CMakePackageConfigHelpers)
 include(GNUInstallDirs)
 
-include(HmdfTargetDataFiles)
+file(GLOB DATA_FILES ${PROJECT_SOURCE_DIR}/data/*.*)
+file(COPY ${DATA_FILES} DESTINATION ${CMAKE_RUNTIME_OUTPUT_DIRECTORY})
```

Comment:
Is this necessary, since you are copying the data below for tests?

Reply:
yes. your example app is built under /bin. while the tests are built under /tests.. Could probably re-direct all builds to the dame directory to avoid having to copy the data twice.

Reply:
Let me know how you want ot proceed. regarding the location of the data files.

I think it's wise to avoid forward/backward slash when pointing out the files as you get more OS-agnostic then.